libs/auth/src/intermediate-scripts/manage-openssl-config

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Requires sudo
+
+# In production, we use the following symlink arrangement:
+#
+# /etc/ssl/openssl.cnf --> /vx/config/openssl.cnf -->
+#   EITHER /etc/ssl/openssl.default.cnf
+#   OR     /vx/code/vxsuite/libs/auth/config/openssl.vx-tpm.cnf
```

Comment:
This symlink arrangement is prepared in the `vxsuite-complete-system` `setup-machine.sh` script: https://github.com/votingworks/vxsuite-complete-system/pull/418